<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NDTDlgClass</class>
 <widget class="QWidget" name="NDTDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDT配准</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;初始变换矩阵&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;NDT参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>511</width>
         <height>171</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>旋转角度(弧度)：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>旋转轴：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="AngleRotation">
           <property name="text">
            <string>0.6931</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="Axis">
           <item>
            <property name="text">
             <string>X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Z</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>W</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>平移向量：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="3">
            <widget class="QLineEdit" name="Z">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;X&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Z&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="Y">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="X">
             <property name="text">
              <string>1.79387</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Y&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>191</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>331</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>网络结构分辨率：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="TransformationEpsilon">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0.01</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="StepSize">
           <property name="text">
            <string>0.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>最小转换差：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="MaximumIterations">
           <property name="text">
            <string>35</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>行搜索最大步长：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>最大迭代次数：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Resolution">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>下采样子叶尺寸:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leaf">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0.2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QLineEdit" name="PathTarget">
       <property name="text">
        <string>target.pcd</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="BrowseTarget">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>选择场景点云：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>选择目标点云：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PathScene">
       <property name="text">
        <string>scene.pcd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="BrowseScene">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>117</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>配准</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
